langchain/docstore/in_memory.py

```diff
@@ -12,10 +12,11 @@ def __init__(self, _dict: Dict[str, Document]):
         """Initialize with dict."""
         self._dict = _dict
 
-    def add(self, texts: Dict[str, Document]) -> None:
+    def add(self, texts: Dict[str, Document], **kwargs: Any) -> None:
```

Comment:
hm i dont love this. will other docstores haave this?

Reply:
i do think its a useful abstraction for docstores -- I overwrite key value stores all the time if there's new information about the key. I could separate it out into a different PR & update the base class too if that makes more sense?